docs/product/support.rst

```diff
@@ -66,42 +66,46 @@ This should be created via a dedicated VM for testing / support activities.
 Chat
 ====
 
-`Olark <https://www.olark.com>`_ is currently used for chat with our customers
-or possible customers.
+`Kayako <https://sftpplus.kayako.com/>`_ is used for chat with our customers
+or possible customers via a widget that is added on the website.
 
-Each chat transcript is automatically forwarded to the Support email list.
+A new conversation is created in Inbox when a visitor initiates the
+conversation and a response is sent via the web platform.
 
-A chat can be transferred and other actions can be taken with the chat via
-Olark `commands <https://www.olark.com/help/commands>`_.
+* An agent should assign a conversation to themselves - either as first
```

Comment:
What is an "agent"? What is "as first responder" ?

If this is early days, do we want to push it to Geck?

Reply:
Kayako uses the word 'agent' so I am taking their term usage.